src/status_im/chat/views/message/message.cljs

```diff
@@ -251,17 +253,41 @@
                          :font  :default}
              (str "+ " (- delivery-statuses-count 3))])]]))))
 
+(defn message-activity-indicator []
+  [react/activity-indicator {:animating true}])
+
+(defn message-not-sent-text [chat-id message-id]
+  [react/touchable-highlight {:on-press (fn [] (if platform/ios?
+                                                 (action-sheet/show {:title   (i18n/label :message-not-sent)
+                                                                     :options [{:label  (i18n/label :resend-message)
+                                                                                :action #(re-frame/dispatch [:resend-message chat-id message-id])}
+                                                                               {:label        (i18n/label :delete-message)
+                                                                                :destructive? true
+                                                                                :action       #(re-frame/dispatch [:delete-message chat-id message-id])}]})
+                                                 (re-frame/dispatch-sync
```

Comment:
I see we have introduced the usage of dispatch-sync, could you explain why in this case is needed and why we should not use dispatch? maybe worth adding a comment somewhere?

Reply:
@cammellos  i was experimenting with dispatch-sync to check if it provides more instant reaction on user taps. Will revert back to async `dispatch` as i don't see much difference.